geode-web-api/src/main/java/org/apache/geode/rest/internal/web/controllers/PdxBasedCrudController.java

```diff
@@ -196,10 +198,9 @@ protected String getRestApiVersion() {
    * Reading data for set of keys
    *
    * @param region gemfire region name
-   * @param keys string containing comma separated keys
    * @return JSON document
    */
-  @RequestMapping(method = RequestMethod.GET, value = "/{region}/{keys}",
+  @RequestMapping(method = RequestMethod.GET, value = "/{region}/**",
```

Comment:
So does doing this now effect any swagger documentation or code generation?

Reply:
swagger shows the endpoint as /v1/{region}/** which seems okay
But if you click try it out it gives you no way (that I could find) to specify a value for the **

Reply:
Feels like there is no real good solution to this problem.